cloudformation/campaign-central-backups.yaml

```diff
@@ -0,0 +1,175 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Campaign central databases backups.
+Parameters:
+  SourceTableName:
+    Description: DynamoDB tableName to backup.
+    Type: String
+    Default: campaign-central-PROD-campaigns
+  DestinationTableName:
+    Description: DynamoDB tableName to write to.
+    Type: String
+    Default: campaign-central-DEV-campaigns
+Resources:
+  CampaignCentralDynamoDBBackup:
+    Type: "AWS::DataPipeline::Pipeline"
+    Properties:
+      Name: CampaignCentralDynamoDBBackup
+      Description: "Pipeline to backup DynamoDB data to S3"
+      Activate: true
+      ParameterObjects:
+        -
+          Id: "myDDBReadThroughputRatio"
+          Attributes:
+            -
+              Key: "description"
+              StringValue: "DynamoDB read throughput ratio"
+            -
+              Key: "type"
+              StringValue: "Double"
+            -
+              Key: "default"
+              StringValue: "0.2"
+        -
+          Id: "myTableName1"
```

Comment:
Can this have a more descriptive name?

Reply:
oh yeah, that's when i thought i was going to put all tables in one cfn template. will update